backend/src/rules/play_card.clj

```diff
@@ -23,14 +24,26 @@
   [game-state player index]
   (modify-hand game-state player (item-remover index)))
 
-(defn ^:private apply-play-card
+(defn ^:private apply-play-card-abilities
+  "Applies the abilities of the played card"
+  [game-state play]
+  (if (nil? (:target play))
+    game-state
+    (alter-card/add-power
+      game-state 
+      (:target play)
+      (get-in game-state [:players (:player play)
+                          :hand (:index play)
+                          :add-power]))))
+
+(defn apply-play-card
```

Comment:
No longer private so we can call it from tests and make them more targeted to what we wanna test.